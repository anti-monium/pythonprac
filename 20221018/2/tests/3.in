:allo msg1 msg2 "allo"+"-"+msg1+"-"+msg2
allo "priv" "yo"
:maximum a b c d f e g max(a,e,f,g)       
maximum 1 2 3 4 5 6 7
allo "meow" "qkrq" 
:summator a b c d e f a+b+c+d
summator 1 2 3 4 5 6
:turboterminator x y z exp(x)+len(y)-7+sin(30)/z*144-max(x,z)+len(x*y)
turboterminator 7 "abc" 4.83984
quit "Have a Good day! God bless you! {}:{}"
