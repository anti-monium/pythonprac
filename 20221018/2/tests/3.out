allo-priv-yo
7
allo-meow-qkrq
10
1077.2362044239103
Have a Good day! God bless you! 5:10
